src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Tab } from './types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  selectedOnTab: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId = tabs[0],
+  selectedOnTab,
+}) => {
+  const selectedTab = tabs.find(({ id }) => id === selectedTabId);
```

Comment:
find can return undefind. Please add to `selected tab` other option for this case

Reply:
When I tried to add it, the picture breaks. Perhaps the reason is the small weight of the images 🤔

Reply:
In progress, looking for another way 🤔